src/components/Card/Card.tsx

```diff
@@ -489,14 +513,14 @@ export default class Card extends React.Component<CardProps, {
               <Button classes={["marketplace-installButton"]}
                 type="circle"
                 // If it is installed, it will remove it when button is clicked, if not it will save
-                label={IS_INSTALLED ? "Remove" : "Save"}
+                label={this.props.type === "app" ? "GitHub" : IS_INSTALLED ? "Remove" : "Install"}
                 onClick={(e) => {
                   e.stopPropagation();
                   this.buttonClicked();
                 }}
               >
                 {/*If the extension, theme, or snippet is already installed, it will display trash, otherwise it displays download*/}
-                {IS_INSTALLED ? <TrashIcon /> : <DownloadIcon />}
+                {this.props.type === "app" ? <GitHubIcon /> : IS_INSTALLED ? <TrashIcon /> : <DownloadIcon />}
```

Comment:
At this point I'm wondering if we make a lookup for these.

Reply:
Will put it on the TODO list